tools/docker/tests/test_flows.sh

```diff
@@ -210,8 +211,11 @@ main() {
 }
 
 VERBOSE=false
-AL_MAC=
-RADIO_WLAN0_MAC=
-RADIO_WLAN2_MAC=
+AL_MAC_agent1=
+AL_MAC_agent2=
+RADIO_WLAN0_MAC_agent1=
+RADIO_WLAN2_MAC_agent1=
+RADIO_WLAN0_MAC_agent2=
+RADIO_WLAN2_MAC_agent2=
```

Comment:
Nitpick: I don't like the naming convention too much. I definitely don't like the mixing of uppercase with lowercase. Since you're changing all these names in this commit anyway, it's a good time to fix them :-)

I propose to keep them lowercase, and like this: mac_agent1, mac_agent1_wlan0

Reply:
OK :)